src/lib/core/overlay/scroll/block-scroll-strategy.spec.ts

```diff
@@ -0,0 +1,133 @@
+import {inject, TestBed, async} from '@angular/core/testing';
+import {ComponentPortal, OverlayModule, BlockScrollStrategy, Platform} from '../../core';
+import {ViewportRuler} from '../position/viewport-ruler';
+
+
+describe('BlockScrollStrategy', () => {
+  let platform = new Platform();
+  let strategy: BlockScrollStrategy;
+  let viewport: ViewportRuler;
+  let forceScrollElement: HTMLElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({ imports: [OverlayModule] }).compileComponents();
+  }));
+
+  beforeEach(inject([ViewportRuler], (_viewportRuler: ViewportRuler) => {
```

Comment:
Don't use `_` for function params; for `inject` I'm fine with just doing something like `vpr`